modules/javafx.graphics/src/main/java/javafx/css/SimpleSelector.java

```diff
@@ -76,32 +77,82 @@ public String getName() {
     /**
      * Gets an immutable list of style-classes of the {@code Selector}.
      * @return an immutable list of style-classes of the {@code Selector}
+     * @deprecated use {@link #getStyleClassNames()} instead
      */
+    @Deprecated(forRemoval = true)
```

Comment:
Given that the class itself is deprecated for removal, marking this method as deprecated is unnecessary.